src/SprykerEco/Zed/AmazonQuicksight/Business/Syncer/QuicksightAssetBundleImportJobSyncer.php

```diff
@@ -0,0 +1,116 @@
+     * @param \SprykerEco\Zed\AmazonQuicksight\Business\Mapper\AmazonQuicksightMapperInterface $amazonQuicksightMapper
+     */
+    public function __construct(
+        AmazonQuicksightApiClientInterface $amazonQuicksightApiClient,
+        AmazonQuicksightConfig $amazonQuicksightConfig,
+        AmazonQuicksightEntityManagerInterface $amazonQuicksightEntityManager,
+        AmazonQuicksightRepositoryInterface $amazonQuicksightRepository,
+        QuicksightAnalyticsRequestValidatorInterface $quicksightAnalyticsRequestValidator,
+        AmazonQuicksightMapperInterface $amazonQuicksightMapper
+    ) {
+        $this->amazonQuicksightApiClient = $amazonQuicksightApiClient;
+        $this->amazonQuicksightConfig = $amazonQuicksightConfig;
+        $this->amazonQuicksightEntityManager = $amazonQuicksightEntityManager;
+        $this->amazonQuicksightRepository = $amazonQuicksightRepository;
+        $this->quicksightAnalyticsRequestValidator = $quicksightAnalyticsRequestValidator;
+        $this->amazonQuicksightMapper = $amazonQuicksightMapper;
+    }
+
+    /**
+     * @return \Generated\Shared\Transfer\QuicksightAssetBundleImportJobTransfer|null
+     */
+    public function findSyncedDefaultQuicksightAssetBundleImportJob(): ?QuicksightAssetBundleImportJobTransfer
+    {
+        $defaultAssetBundleImportJobId = $this->amazonQuicksightConfig->getDefaultAssetBundleImportJobId();
+
+        $quicksightAssetBundleImportJobTransfer = $this->amazonQuicksightRepository->findQuicksightAssetBundleImportJobByJobId(
+            $defaultAssetBundleImportJobId,
+        );
+
+        if (!$this->quicksightAnalyticsRequestValidator->isAssetBundleInitializationInProgress($quicksightAssetBundleImportJobTransfer)) {
+            return $quicksightAssetBundleImportJobTransfer;
+        }
+
+        $quicksightDescribeAssetBundleImportJobResponseTransfer = $this->amazonQuicksightApiClient
+            ->describeAssetBundleImportJob($defaultAssetBundleImportJobId);
+
+        if ($quicksightDescribeAssetBundleImportJobResponseTransfer->getJobStatus() === null) {
+            $quicksightDescribeAssetBundleImportJobResponseTransfer->setJobStatus(
+                $quicksightAssetBundleImportJobTransfer->getStatusOrFail(),
+            );
+        }
+
+        $quicksightAssetBundleImportJobTransfer = $this->amazonQuicksightMapper
+            ->mapQuicksightDescribeAssetBundleImportJobResponseTransferToQuicksightAssetBundleImportJobTransfer(
+                $quicksightDescribeAssetBundleImportJobResponseTransfer,
+                $quicksightAssetBundleImportJobTransfer,
+            );
+
+        if ($quicksightAssetBundleImportJobTransfer->getStatus() === static::ASSET_BUNDLE_IMPORT_JOB_STATUS_SUCCESSFUL) {
+            $quicksightAssetBundleImportJobTransfer->setIsInitialized(true);
+        }
+
+        return $this->amazonQuicksightEntityManager
+            ->updateQuicksightAssetBundleImportJob($quicksightAssetBundleImportJobTransfer);
```

Comment:
Looks like a SOLID violation. 
Please, consider moving the code where we call `describeAssetBundleImportJob` and update the `QuicksightAssetBundleImportJob` to a separate `Updater` class.

Reply:
describeAssetBundleImportJob is not responsibility of Updater. For me this is part of the synchronisation. The line where we call the entity manager could be in updater but it doesn't make sense to create a 1-line method.